nx-X11/programs/Xserver/hw/nxagent/NXevents.c

```diff
@@ -4335,6 +4338,7 @@ ProcGrabKeyboard(ClientPtr client)
 			KeyPressMask | KeyReleaseMask, &rep.status);
     if (result != Success)
 	return result;
+    memset(&rep, 0, sizeof(xGrabKeyboardReply));
```

Comment:
Please move this up to line 4326, as done in `nx-X11/programs/Xserver/dix/events.c`.